File: Python/Step_6_For_Loops.py
for a in range(10):
    print(f"Count: {a}")
    cond = input('Do you want me to stop?[y/n]: ')
    if cond == 'y':
        break


print("Second Loop Counting Starts Here............")
for a in range(0, 10, 2):
    print(f"Count: {a}") 
    
# for the second loop, the format is
# for a[this gets increased in every loop] in range(start, end, steps):

File: C Program/Step_6_For_Loops.c
#include <stdio.h>
#include <string.h>

int main(){

    for (int a = 0; a<=10; a++){

        printf("Count: %d\n", a);
        char cond[10];
        printf("Do you want me to stop?[y/n]: ");
        scanf("%s", cond);

        if (strcmp(cond, "y") == 0){

            break;

        }
    }

    return 0;
}